src/ec/packages/ec-component-message/message.story.js

```diff
@@ -15,6 +19,40 @@ import dataWarning from './demo/data--warning';
 import message from './ecl-message.html.twig';
 import notes from './README.md';
 
+const iconsList = {};
+iconsList.none = null;
+
+notifIcons.forEach(icon => {
+  iconsList[icon] = icon;
+});
+
+const PrepareMessage = data => {
+  const name = select(
+    'icon.name',
+    iconsList,
+    data.icon.name,
+    buttonLabels.required
+  );
+  if (name !== null) {
+    getIconKnobs(data, name, 'notifications', 'l');
+  }
+
```

Comment:
I don't see the color acting in a anyway and i don't think we should give the chance to edit the transform either, here it's one icon per variant, no "options" whatsoever, so pass also the other parameters, i'd sy, to the function giving you the icon knobs.